# Description: lower CPU optimization level to i486, which is the lowest
#  Debian supports
# Author: Damyan Ivanov <dmn@debian.org>
# Forwarded: not-needed
--- a/builds/posix/prefix.linux
+++ b/builds/posix/prefix.linux
@@ -20,7 +20,7 @@
 
 COMMON_FLAGS=-ggdb -DFB_SEND_FLAGS=MSG_NOSIGNAL -DLINUX -pipe -MMD -fPIC -fmessage-length=0 -fno-delete-null-pointer-checks
 CXXFLAGS+=-std=gnu++03
-OPTIMIZE_FLAGS=-O3 -march=i586 -mtune=i686 -fno-omit-frame-pointer
+OPTIMIZE_FLAGS=-O3 -march=i686 -mtune=i686 -fno-omit-frame-pointer
 WARN_FLAGS=-Wall -Wno-switch -Wno-parentheses -Wno-unknown-pragmas -Wno-unused-variable -Wno-narrowing
 
 PROD_FLAGS=$(COMMON_FLAGS) $(OPTIMIZE_FLAGS)
--- a/builds/posix/prefix.freebsd
+++ b/builds/posix/prefix.freebsd
@@ -20,7 +20,7 @@
 
 COMMON_FLAGS=-ggdb -DFB_SEND_FLAGS=MSG_NOSIGNAL -DFREEBSD -pipe -MMD -fPIC -fmessage-length=0 -fno-delete-null-pointer-checks
 CXXFLAGS+=-std=gnu++03
-OPTIMIZE_FLAGS=-O3 -march=i586 -mtune=i686 -fno-omit-frame-pointer
+OPTIMIZE_FLAGS=-O3 -march=i686 -mtune=i686 -fno-omit-frame-pointer
 WARN_FLAGS=-Wall -Wno-switch -Wno-parentheses -Wno-unknown-pragmas -Wno-unused-variable -Wno-narrowing
 
 PROD_FLAGS=$(COMMON_FLAGS) $(OPTIMIZE_FLAGS)
